edx_sga/sga.py

```diff
@@ -57,69 +57,69 @@ class StaffGradedAssignmentXBlock(XBlock):
     icon_class = 'problem'
 
     display_name = String(
-        default='Staff Graded Assignment', scope=Scope.settings,
-        help="This name appears in the horizontal navigation at the top of "
-             "the page."
+        default= "Staff Graded Assignment", scope=Scope.settings,
+        help= _("This name appears in the horizontal navigation at the top of ") +
+              _("the page.")
```

Comment:
You can't split a sentence for translation.  Keep it all as one string, even if it's two string literals implicitly concatenated.